Automattic/jetpack
# VideoPress: TypeScriptify code

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: TypeScriptify code. Janitorial tasks.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Everything should work as expected.
* So far, you should check "[Privacy](https://github.com/Automattic/jetpack/pull/27401) and [Rating](https://github.com/Automattic/jetpack/pull/27415)" panel.
